exercises/e-data02-derive-master.ipynb

```diff
@@ -0,0 +1,866 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #8.            #&gt;&gt; gr.tf_mutate(price_cov = gr.round(DF["price_cov"], 3)) &lt;- Expected  round call
</pre><p>Wow, that's embarassing... I'm fixing this in a <a href="https://github.com/zdelrosario/py_grama/pull/171" target="_blank">PR</a> .</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/zdelrosario/evc-course/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>